config/locales/notifications/en.yml

```diff
@@ -20,6 +20,21 @@ en:
       1: >
         New feedback has been added to iteration (#%{iteration_idx}) of your solution to <strong>%{exercise_title}</strong> in <strong>%{track_title}</strong>.
         We %{emphasis}recommend taking a look.
+
+    onboarding_product:
+      1: >
+        TODO
+
+    onboarding_community:
+      1: >
+        Community is at the heart of Exercism. Try our forums and mentoring, get involved on YouTube and Twitch.
+        Check the community section for more info!
+
+    onboarding_insiders:
+      1: >
+        Support our goal: Give everyone an equal chance to learn and master programming.
+        Check the donate section for more info!
+
```

Comment:
@ihid You'll want to remove these if these should just be email-only

Reply:
Thanks. One should be email only. The other two should be notifications. I've updated things to check accordingly.